### PR TITLE
set up postgres database connection [Finishes #163416658]

### PR DESCRIPTION
**Pull request by Mastula77**
I have set up database connection and successfully connected to PostgreSQL and updated the app file
@Rhytah, @Habibsentongo, @wasswa-derick, @nicholusmuwonge, @richien

### CODE REVIEW

**Review by NicholusMuwonge — COMMENTED** [review]
looks good , go ahead and merge up, time is thinning down

**Comment by HabibSentongo** [comment]
good to see you have fully overcome this big blocker you had, speed up your next steps a little bit

File: app.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
 from api.views import app
+from api.dbconnect import Dbconnection
+
+testss = Dbconnection()
```
Comment (Rhytah) [inline app.py:4]: Consider changing object name to reflect relevance to class. For instance, `dbconnect=Dbconnection()`

**Comment by Rhytah** [comment]
Good work, the connection is successful, you should be able to start your database queries now.
Move on to that and view outputs

**Pull request merged**